src/SearchField/SearchField.scss

```diff
@@ -1,83 +1,33 @@
-$search-field-clear-btn-width: 30px;
-$search-field-border-radius: 30px;
-
-.search-field {
+.pgn-searchfield {
     transition: $input-transition;
-    border-radius: $search-field-border-radius;
-    box-shadow: $input-box-shadow;
-    background: #ffffff;
-
-    .form-group {
-        margin-bottom: 0;
-    }
-
-    .input-group {
-        flex: 1 1 0;
-
-        //clear browser 'x' from Internet Explorer
-        input[type=search]::-ms-clear {
-          display: none;
-          width : 0;
-          height: 0;
-        }
-        input[type=search]::-ms-reveal {
-          display: none;
-          width : 0;
-          height: 0;
-        }
-
-        //clear browser 'x' from Chrome
-        input[type="search"]::-webkit-search-decoration,
-        input[type="search"]::-webkit-search-cancel-button,
-        input[type="search"]::-webkit-search-results-button,
-        input[type="search"]::-webkit-search-results-decoration {
-          display: none;
-        }
-    }
 
-    label {
-        margin: 0;
-        padding-left: 20px;
-        padding-right: 10px;
-    }
-
-    button:last-child {
-        border-radius: $search-field-border-radius;
-    }
-
-    &.focused {
-        border-color: $blue !important;
+    &.has-focus {
         box-shadow: $input-focus-box-shadow;
     }
 
-    .input {
-        background: none;
-        border: 0;
-        box-shadow: none;
-        padding-left: 0;
-        padding-right: 0;
-        margin-top: 0.15rem;
-
+    .form-control {
         &:focus {
-            background: none;
+            box-shadow: none;
```

Comment:
Can you add **outline: transparent** to this? It makes the outline style visible in Windows High Contrast mode.  Otherwise, it'll be visible here(if not suppressed somewhere else), or outline: none would remove it as well as the box-shadow in HC mode, leaving no focus styling.

Reply:
@wittjeff `outline: transparent` does already exist on Bootstrap `.form-control` input elements:

![image](https://user-images.githubusercontent.com/2828721/80373043-763b6d00-8862-11ea-91ac-aa64eeb765b6.png)